packages/shell-api/src/collection.ts

```diff
@@ -142,9 +142,12 @@ export default class Collection extends ShellApiWithMongoClass {
    *
    * @returns {Promise} The promise of aggregation results.
    */
+  async aggregate(pipeline: Document[], options: Document & { explain?: undefined | false }): Promise<AggregationCursor>
+  async aggregate(pipeline: Document[], options: Document & { explain: true }): Promise<Document>
```

Comment:
explain: false` is `explain: 'queryPlanner'`, I think that’s not what you are expecting here?

```suggestion
  async aggregate(pipeline: Document[], options: Document & { explain?: never }): Promise<AggregationCursor>
  async aggregate(pipeline: Document[], options: Document & { explain: ExplainVerbosityLike }): Promise<Document>`
```

Reply:
I think that’s very surprising behavior, and not what we are implementing here…

Reply:
I see that the _server_ accepts the verbosity... I'll change it accordingly 👍 Sad that the docs don't show that 😞